index.js

```diff
@@ -15,16 +17,30 @@ const body = await new Promise((resolve, reject) => {
 	});
 });
 
-const signed = body
+const signedUsers = [...body
 	.match(/^\|.+\|$/mg)
 	?.slice(2)
-	.map(u => u.split('|')[2].trim());
+	.map(u => u.split('|')[2].trim()),
+	'dependabot[bot]'
+];
```

Comment:
Couldn't you just done it the old school way and pushed dependabot onto the array? :)